src/shared/components/FormComponents/EditorController/Editor/Editor.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import debounce from 'lodash.debounce';
+import 'md-editor-rt/lib/style.css';
+
+import {
+  CharacterCounter,
+  FooterMessage,
+  LANGUAGE_BY_DEFAULT,
+} from 'shared/components/MarkDownEditor';
+import { Spinner, SpinnerUiType } from 'shared/components/Spinner';
+import { StyledFlexColumn, StyledFlexSpaceBetween, theme } from 'shared/styles';
+import { getSanitizedContent } from 'shared/utils/forms';
+import { CHANGE_DEBOUNCE_VALUE } from 'shared/consts';
+
+import { StyledMdEditor } from './Editor.styles';
+import { getCustomIcons, getDefToolbars, getToolbars } from './Editor.utils';
+import { EditorProps } from './Editor.types';
+
+export const Editor = ({
+  editorId,
+  editorRef,
+  value,
+  onChange,
+  onInsert,
+  onLoading,
+  onFileExceeded,
+  onIncorrectFileFormat,
+  uiType,
+  error,
+  disabled,
+  isLoading,
+  withDebounce,
+  'data-testid': dataTestid,
+}: EditorProps) => {
+  const { t } = useTranslation('app');
+  const [inputValue, setInputValue] = useState(value ?? '');
+
+  const handleDebouncedChange = useCallback(
+    debounce((v) => onChange(v), CHANGE_DEBOUNCE_VALUE),
```

Comment:
pls change name "v" to more understandable

Reply:
done